### PR TITLE
feat(*) examples for ECS Universal deployments

### PR DESCRIPTION
**Pull request by nickolaev**
### Summary

The example CloudFormaion scripts for deploying Kuma in Universal mode on ECS.

### CODE REVIEW

**Comment by subnetmarco** [comment]
@nickolaev can we also submit a `PR` into `kuma-website` for a new `ecs` deployment distribution that explains the steps required to deploy on any ECS cluster? Once that PR is done, I would like to review it before merging it.

File: examples/ecs/README.md [hunk 1]
```diff
@@ -126,6 +136,7 @@ aws cloudformation deploy \
     --template-file workload.yaml \
     --parameter-overrides \
       DesiredCount=2 \
+      DPToken="<DP_TOKEN_VALUE> \
```
Comment (nickolaev) [inline examples/ecs/README.md:139]: We need a closing "

Comment (lobkovilya) [reply examples/ecs/README.md]: Thanks!

File: examples/ecs/Makefile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+ecs/ssh-client/deploy:
+	aws cloudformation deploy \
+		--capabilities CAPABILITY_IAM \
+		--stack-name ssh-client \
+		--template-file workload.yaml \
+		--parameter-overrides \
+		  DesiredCount=1 \
+		  DPToken=${DATAPLANE_TOKEN_SSHCLIENT} \
+		  WorkloadImage=docker.io/rastasheep/ubuntu-sshd:14.04 \
```
Comment (nickolaev) [inline examples/ecs/Makefile:58]: Can we use `Image: sickp/alpine-sshd:latest`

**Pull request merged**